tests/client.py

```diff
@@ -833,6 +835,107 @@ def create(self, stream, company_ids=[], subscriptions=[], times=1):
+            "groupName": "contactinformation",
+            "type": "string",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 6,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "custom_number",
+            "label": "A New Number Custom Property",
+            "description": "A new number property for you",
+            "groupName": "contactinformation",
+            "type": "number",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 7,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "custom_date",
+            "label": "A New Date Custom Property",
+            "description": "A new date property for you",
+            "groupName": "contactinformation",
+            "type": "date",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 9,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "custom_datetime",
+            "label": "A New Datetime Custom Property",
+            "description": "A new datetime property for you",
+            "groupName": "contactinformation",
+            "type": "datetime",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 10,
+            "options": [
+            ]
+        }
+        data.append(deepcopy(property))
+
+        property = {
+            "name": "multi_pick",
+            "label": "multi pick",
+            "description": "multi select picklist test",
+            "groupName": "contactinformation",
+            "type": "enumeration",
+            "fieldType": "checkbox",
+            "hidden": False,
+            "options": [
+              {
+                "label": "Option A",
+                "value": "option_a"
+              },
+              {
+                "label": "Option B",
+                "value": "option_b"
+              },
+              {
+                "label": "Option C",
+                "value": "option_c"
+              }
+            ],
+            "formField": True
+        }
+        data.append(deepcopy(property))
+        # generate a contacts record
+
+        for current_data in data:
+            try:
+                response = self.post(url, current_data)
+                LOGGER.info("response is %s", response)
+            # Setting up the property is a one time task, If exception occurs because, it already exists, ignore
+            except Exception as DataAlreadyExistsExcp:
+                LOGGER.info("Data already exists for %s", current_data)
+                if '409' in str(DataAlreadyExistsExcp):
+                    pass
+                else:
+                    response.raise_for_status()
```

Comment:
I don't think I"m communicating well enough.  doing `except Exception` is a no-no as this is a very broad exception.  We need to figure out the specific exception that is being thrown and catch that.  To determine what exception is being thrown you can remove the try/except and run the test to see what exception is raised. Catch that one.

Reply:
> [Exception](https://docs.python.org/3/library/exceptions.html#Exception) can be used as a wildcard that catches (almost) everything. However, it is good practice to be as specific as possible with the types of exceptions that we intend to handle, and to allow any unexpected exceptions to propagate on.

https://docs.python.org/3/tutorial/errors.html

Reply:
Sorry Harrison - I think I got it right now :) Now handling it as requests.exceptions.HTTPError